static/app/components/events/eventStatisticalDetector/regressionMessage.tsx

```diff
@@ -0,0 +1,61 @@
+import {DataSection} from 'sentry/components/events/styles';
+import Link from 'sentry/components/links/link';
+import {t, tct} from 'sentry/locale';
+import {Event} from 'sentry/types';
+import {formatPercentage} from 'sentry/utils/formatters';
+import useOrganization from 'sentry/utils/useOrganization';
+import {normalizeUrl} from 'sentry/utils/withDomainRequired';
+import {
+  DisplayModes,
+  transactionSummaryRouteWithQuery,
+} from 'sentry/views/performance/transactionSummary/utils';
+
+type EventStatisticalDetectorMessageProps = {
+  event: Event;
+};
+
+function EventStatisticalDetectorMessage({event}: EventStatisticalDetectorMessageProps) {
+  const organization = useOrganization();
+
+  const transactionName = event?.occurrence?.evidenceData?.transaction;
+  const transactionSummaryLink = transactionSummaryRouteWithQuery({
+    orgSlug: organization.slug,
+    transaction: transactionName,
+    query: {},
+    projectID: event.projectID,
+    display: DisplayModes.TREND,
+  });
+  const detectionTime = new Date(0);
+  detectionTime.setUTCSeconds(event?.occurrence?.evidenceData?.breakpoint);
+
+  // TODO: This messaging should respect selected locale in user settings
+  return (
+    <DataSection>
+      <div style={{display: 'inline'}}>
+        {tct(
+          '[detected] Based on the transaction [transactionName], there was a [amount] increase in duration (P95) around [date] at [time] UTC. Overall operation percentage changes indicate what may have changed in the regression.',
```

Comment:
Yeah, that's my understanding of what it's pointing to as well. I think it's a bit difficult to make the connection because in the mock ups the Operation Breakdown is actually below the chart. I wonder if we could raise moving the text to below the chart to connect them better, but we could tackle that separately